datasetinsights/configs/faster_rcnn_groceries_real.yaml

```diff
@@ -42,3 +42,9 @@ optimizer:
 pretrained: False
 pretrained_backbone: True
 synchronize_metrics: True
+tracker:
```

Comment:
The code should not fail without changing the original yaml file. So, why is this change required?